crates/biome_js_formatter/tests/specs/prettier/js/if/non-block.js.snap

```diff
@@ -0,0 +1,44 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: js/if/non-block.js
+---
+# Input
+
+```js
+if (foo) for (i = 2; i > 0; i--) console.log(i); // foo
+else bar();
+
+if (foo) do { console.log(i) } while(i--) // foo
+else bar();
+
+```
+
+
+# Prettier differences
+
+```diff
+--- Prettier
++++ Biome
+@@ -5,6 +5,5 @@
+ if (foo)
+   do {
+     console.log(i);
+-  } while (i--);
+-// foo
```

Comment:
This mismatch is caused by reformat issue with Prettier. [playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAZgAgBTohASkwBMJNhNIoBnCAGzgDo6IBzbVIgX0wHcALVAw4BaEUQD0EzHggAdKHDrU4mAEYBDAE7YCAbgUKpmGP1UQArjAAOVzBCynV6VFuowZELQFsNMBRg4sgQKmCRkwKFhlNC0DMxsHPpRPAJCqqLiepjGsgpKKuraunogADQgENYwqLHIoNpaELwACtoI1MggGnS8GgCenRVqWhpgANZwMADKGt5wADKoisjoPSrDoxNT09Zjy6zIMFoWcBUq3qhHJ2cgcAAe1nBaqPOwPQAqz1DaqHCdSDWylu1AODAAihYIPBVutbgAraj3aZguCQ6FwWHAioARyh8BaTWsAO61BEijgxEp5RAxw0QgOAGEIN5fF0enQaaCoKwGABBGDHVBqKxwFrPJYrQFwir8GDeOgAdUE8GoezAcGmHVQNQAbjr+l0wNQhiBdacAJJQKmwaZgF7VPnW6YwfoMLEbEDWJoqRWjaxdb3-Z66zEVZYqLQwQkaVhs6XYr3aSNdTRqJQ073LGCK1DEUzIAAcAAYKlo4HjXGLRnGNB7bjANGpc-n+MgAEwVCwqD5NgFAz1wbzp4hU4gLDQ8iyxuAAMS8vkFB3ZVggIC4XCAA)